openvdb_points/tools/AttributeSet.h

```diff
@@ -305,6 +337,7 @@ class AttributeSet::Descriptor
     size_t insert(const std::string& name, const NamePair& typeName);
     NameToPosMap                mNameMap;
     std::vector<NamePair>       mTypes;
+    NameToPosMap                mGroupMap;
```

Comment:
My current understanding is that there is 1 byte per point in each GroupAttributeArray. Each byte encodes the membership of up to 8 groups. Which is why we have a unit test creating 3 GroupAttributeArrays for 17 groups. So the offset is just the group's bit index (from 0...NUM_GROUPS). You then use the groupIndex method to obtain a pair containing the index of the GroupAttributeArray (e.g. an index into the descriptor vector), and the bit position of the group within this GroupAttributeArray. Is this correct?